src/Workspaces/Core/Portable/Formatting/Engine/TokenStream.cs

```diff
@@ -486,8 +491,17 @@ public bool IsFirstTokenOnLine(SyntaxToken token)
             return IsFirstTokenOnLine(previousTokenWithIndex, tokenWithIndex);
         }
 
+        private bool IsFirstTokenOnLine(in TokenData tokenData)
+        {
+            Contract.ThrowIfTrue(tokenData.Token.RawKind == 0);
+
+            var previousTokenWithIndex = tokenData.GetPreviousTokenData();
+
+            return IsFirstTokenOnLine(in previousTokenWithIndex, in tokenData);
```

Comment:
thanks for the `in`.  i personally would prefer they always be there.

@sharwell any thoughts on a code-style that requires this sort of thing?